pkg/runctl/run_manager.go

```diff
@@ -568,6 +568,28 @@ func (c *runManager) createTektonTaskRun(ctx *runContext) error {
 	return err
 }
 
+func addTektonTaskRunParamsForImage(
+	pipelineRun k8s.PipelineRun,
+	tektonTaskRun *tekton.TaskRun,
+) {
+	spec := pipelineRun.GetSpec()
+	image := spec.JenkinsfileRunnerImage
+	if image != nil {
```

Comment:
variable name `image` should be renamed to `jfrImageSpec` or `imageSpec`

Reply:
renamed to more generic jfrSpec